src/utils/validApiKey.ts

```diff
@@ -11,7 +11,7 @@ export const validateApiKey = async (apiKey: string): Promise<boolean> => {
       },
     })
     return true
-  } catch {
+  } catch (error) {
```

Comment:
```suggestion
  } catch {
```

This is giving out "error" not used error for me

Reply:
yes, we can remove error parameter and just retrun false